.travis.yml

```diff
@@ -10,11 +10,31 @@ install:
   - python3 -m pip install pytest
 
 script:
+  #
+  # We first record the version of everything we use as a baseline
+  # for future regressions introduced by these tools being upgraded.
+  #
+  - yapf --version
```

Comment:
An alternative solution would be to (perhaps at some later point) use specific version of the packages, so we have a more reproducible developer environment. There's tools that can help us achieve this, if we wanted to go that route.

Reply:
Oh yeah that would be preferable indeed. I'll merge this for now and we can take out the above part from Travis when we introduce that change.